### PR TITLE
Sale last price porting

### PR DESCRIPTION
**Pull request by abaghirli**

### CODE REVIEW

**Comment by rafaelbn** [comment]
Hi @abaghirli , thanks for this contribution. When you say "porting" what do you mean?

I guess the module you are migrating is https://github.com/OCA/sale-workflow/tree/10.0/sale_last_price_info

But you have change name, why? It was long time ago discussed in : https://github.com/OCA/sale-workflow/issues/53

Please review and fix. Also is needed to increase code coverage please. Thank!

**Comment by yvaucher** [comment]
@rafaelbn this module was there in 7.0 branch:

https://github.com/OCA/sale-workflow/tree/7.0/last_sale_price

**Comment by yvaucher** [comment]
Those 2 modules are not exactly the same

**Comment by yvaucher** [comment]
Differences between the two modules:

**sale_last_price_info**

Show the last customer to which the product was sold with price and date on the product form.

I'm not sure what is the use case of this module (README.rst of sale_last_price_info needs more info).

**last_sale_price**

Show the last price, date and quantity sold to the same partner on the sale order line.
This module intends to give a view to the seller to make sure he set a price according to the last sale on a customer basis in order to be coherent between 2 sales.

----

So I think this are 2 different use case. Maybe both module should get a better name. Like last_sale_price_per_partner for this one and last_sale_price_per_product ?


@oihane @pedrobaeza  @alfredoavanzosc your opinion is welcome on this.

**Comment by pedrobaeza** [comment]
I prefer to have `sale` as the first word, but the rest of the renaming seems good to me.

**Comment by rafaelbn** [comment]
👍 to :

- sale_last_price_per_partner for this one and
- sale_last_price_per_product for https://github.com/OCA/sale-workflow/tree/10.0/sale_last_price_info

About this module I think should be improved coverage

**Comment by alfredoavanzosc** [comment]
:+1:

**Comment by yvaucher** [comment]
@abaghirli Please rename this module to sale_last_price_per_partner.

Plus add a sentence on README.rst to say it is the former last_sale_price module.

File: sale_last_price_per_partner/test/__init__.py [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Camptocamp
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from . import test_last_sale_price_per_partner
```
Comment (oihane) [inline sale_last_price_per_partner/test/__init__.py]: Newline missing

Comment (abaghirli) [reply sale_last_price_per_partner/test/__init__.py]: aren't init files ignored?
lint check passed ok

Comment (oihane) [reply sale_last_price_per_partner/test/__init__.py]: Yes, lint check ignores `__init__.py` files, but it is a good practice to add a newline in the files.

File: sale_last_price_per_partner/README.rst [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+Credits
+=======
+
+Contributors
+------------
+* Esther Martín <esthermartin@avanzosc.es>
+* Pedro M. Baeza <pedro.baeza@serviciosbaeza.com>
+* Ana Juaristi <anajuaristi@avanzosc.es>
+* Oihane Crucelaegui <oihanecrucelaegi@avanzosc.es>
```
Comment (yvaucher) [inline sale_last_price_per_partner/README.rst]: As far as I know only you and me worked in terms of code contribution on this module.

Well it's arguable for the reviewers as they contributes in a way. Nevertheless, and I speak for myself here, I prefer not to have the name of a code reviewer here as he might knows the module less than it's creators thus it doesn't make a good choice to have him in contacts.

**Review by yvaucher — APPROVED** [review]
Appart from contributor list which seems strange to me, it looks good.

**Comment by rousseldenis** [comment]
@abaghirli Is this still applicable ?

**Comment by rousseldenis** [comment]
@abaghirli Can you rebase ?

**Comment by yvaucher** [comment]
@rousseldenis @abaghirli doesn't work at Camptocamp anymore. This PR needs to be overseeded.

It seemed almost good though but it has been a while.

**Comment by rousseldenis** [comment]
Closing this as it won't be fixed. Feel free to reopen it if needed

**Pull request closed without merging**